osu.Game.Rulesets.Osu/Difficulty/Skills/Speed.cs

```diff
@@ -20,20 +23,116 @@ public class Speed : Skill
         private const double pi_over_4 = Math.PI / 4;
         private const double pi_over_2 = Math.PI / 2;
 
+        private const double streamaimconst = 2.42;
+
+        //  private const double stdevconst = 0.149820;
+
+        private int sdsplitcounter = 0;
+        private int sdsplitcounter2 = 0;
+        // private int runcheck1 = 0;
+
+        private double sdstrainmult = 0;
+        // private double sdstrainmult2;
+
+        // private double sdstrainmult2;
```

Comment:
Commented code is not permitted. It should be uncommented or removed.

Reply:
Fixed in e11de4b.